packages/gallery/src/components/gallery/index.js

```diff
@@ -76,8 +76,10 @@ export default class BaseGallery extends React.Component {
         }
       }
     }
+    else if(galleryType === undefined || galleryLayout === LAYOUTS.GRID) {
```

Comment:
Why is this needed?
It looks like this condition is contained within the previous

Reply:
I agree it is not too elegant to have this condition twice (in the switch-case and here) but we somehow need to access to the "GridGallery" component. 

By default the GalleryComponent is "ProGallery",
and if the bluePrints aren't defined we won't access the switch-case at all --> so no chance to use the "GridGallery".

Reply:
This can run inside the LeanGallery
`addLayoutStyles(addPresetStyles`

Reply:
styles = {...defaultStyles, ...styles}
      formattedStyles = processLayouts(addPresetStyles(styles));